lib/modules/test.go

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
I was looking for a more descriptive file name, but `modules_test.go` is a legitimate test file, so maybe `export_test.go`? WDYT?

Reply:
@codingllama I don't think it can be put in a `_test` file and be imported by other packages. I chose `test.go` because we have other files in the repo with the same name that provides similar test-only functionality.

```shell
./lib/srv/db/mongodb/test.go
./lib/srv/db/common/test.go
./lib/srv/db/postgres/test.go
./lib/srv/db/mysql/test.go
```

Reply:
@codingllama I didn't think it was possible for other packages to import these symbols if it is in a `_test` file. Every time I've tried that it didn't work. It also must reference non-exported symbols from `modules`, so it can't be in `modules_test` package.

I chose `test.go` because we have other files in the repo with the same name that provides similar test-only functionality.

```shell
./lib/srv/db/mongodb/test.go
./lib/srv/db/common/test.go
./lib/srv/db/postgres/test.go
./lib/srv/db/mysql/test.go
```